frontend/src/container/tables/FavoritesTable/FavoriteLineComponent.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import {
+  tracimTableLib
+} from 'tracim_frontend_lib'
+
+const { GenericTracimTableLine, RowWrappers } = tracimTableLib
+
+const FavoriteLineComponent = props => {
+  const Wrapper = RowWrappers.ListItemRowWrapper
+  return (
+    <Wrapper
+      customClass='favoriteTable__row'
+      contentType={props.contentType}
+      {...props.rowData.original}
```

Comment:
This is not wrong, but a point of attention here is that if `props.rowData.original` has `customClass` or `contentType`, these properties will be rewritten and the values declared here will not be used, a good practice if you want to guarantee the values is to invert the order.
```suggestion
      {...props.rowData.original}
      contentType={props.contentType}
      customClass='favoriteTable__row'
```